tests/test_users.py

```
@@ -285,10 +287,7 @@ def test_delete_user(self):
             self.assertEqual(response[1].code().name, "OK")
             self.user_id = None  # User deleted successfully
         except Exception as e:
-            # If delete fails, that's also acceptable for this test
-            # as the backend might have constraints
-            print(f"Delete user failed (expected in some cases): {str(e)}")
-            self.user_id = None
+            raise e
```

Ruff (0.15.2)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)